src/controllers/handlers.ts

```diff
@@ -268,6 +273,13 @@ export const estateRequestHandler = async (context: {
     return { status: 503, body: 'Not ready' }
   }
 
+  if (id !== null) {
```

Comment:
same question as above here, probably a `null` id should be invalid at this point too

Reply:
true, removed!